README.md

```diff
@@ -1,4 +1,6 @@
 # crypto_policies
+![CI Testing](https://github.com/linux-system-roles/crypto_policies/workflows/tox/badge.svg)
```

Comment:
This still shows "Not found". When the icon is supposed to be ready? After this will be merged and we will submit another PR?

Additionally, I noticed that icons converted from markdown to html are somehow weird when packaged into rhel-system-roles package. Not sure if it is something that we need to address now and here as having the build status here makes sense, but it has less sense in the packaged readme, especially if they are rendered wrongly.

Reply:
I have no idea what the badge should look like.  If you have a working example I would be grateful.  I just copied and pasted what the Action link generated.

Reply:
And yes - the icon will not be ready until after this PR is merged (and it may take a few minutes after that)